README.rst

```diff
@@ -20,19 +19,152 @@ StreamStats
 
 Python package for interfacing with the USGS StreamStats API.
 
+- Free software: MIT license
+- Documentation: https://streamstats-python.readthedocs.io/en/latest/
+- GitHub: https://github.com/earthlab/streamstats
+
+Features
+~~~~~~~~~
 
-* Free software: MIT license
-* Documentation: https://streamstats-python.readthedocs.io/en/latest/
+- Plot the GeoJSON of a watershed containing a spatial point in the United States
+- Find available basin characteristics of an identified watershed
+- Find the hydrologic unit code (HUC) of an identified watershed
 
 
-Features
---------
+View Example StreamStats Applications in Our Documentation Gallery
+-------------------------------------------------------------------
+Check out our `vignette gallery <https://streamstats-python.readthedocs.io/en/latest/gallery_vignettes/index.html>`_
+for applied examples of using StreamStats.
+
+
+Installation
+~~~~~~~~~~~~~
+Stable release
+--------------
+To install StreamStats via ``pip`` use:
+
+.. code-block:: python
```

Comment:
I don't think it matters to much, but I believe this code block should be console, not python.